tutorials/in-app/knightPlatformer.json

```diff
@@ -0,0 +1,264 @@
+{
+  "id": "knightPlatformer",
+  "editorSwitches": {
+    "Start": {
+      "editor": "Scene",
+      "scene": "level1"
+    }
+  },
+  "availableLocales": [
+    "en"
+  ],
+  "initialProjectData": {
+    "playerHitBox": "KnightHitBox",
+    "tiles": "Tiles",
+    "level1": "Level 1"
+  },
+  "flow": [
+    {
+      "id": "Start",
+      "elementToHighlightId": "#toolbar-preview-button",
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "This game is a Platformer where the player must get the key then read the door to open it.Let see the current game state.\n\nClick on the **preview** button to play."
+          }
+        }
+      },
+      "nextStepTrigger": {
+        "previewLaunched": true
+      }
+    },
+    {
+      "tooltip": {
+        "standalone": true,
+        "description": {
+          "messageByLocale": {
+            "en": "The knight cannot move, the behavior Platformer character on the **$(playerHitBox)** is missing.\n\nLet's see how to add it to the object."
+          }
+        }
+      },
+      "nextStepTrigger": {
+        "clickOnTooltipButton": {
+          "messageByLocale": {
+            "en": "Next"
+          }
+        }
+      }
+    },
+    {
+      "elementToHighlightId": "objectInObjectsList:playerHitBox",
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Let's add movement to our knight with the help of a **behavior**.\n\nClick on the 3 dot menu, or right Click on **$(playerHitBox)**, and select **Edit behaviors**."
+          }
+        }
+      },
+      "nextStepTrigger": {
+        "presenceOfElement": "#object-editor-dialog"
+      }
+    },
+    {
+      "elementToHighlightId": "#behaviors-tab",
```

Comment:
This step should be marked skippable: it avoids a glitch where the tooltip is first displayed on the tab even though the user is already on the right tab.

Reply:
With my new PR, you will be able to use the meta step Add behavior that handles this kind of things

Reply:
Also, this should be marked "isOnClosableDialog" too